src/test/java/uk/gov/hmcts/opal/controllers/DefendantAccountControllerTest.java

```diff
@@ -103,7 +101,7 @@ public void testGetDefendantAccounts_Success() {
 
     }
 
-    @Test
+    /*@Test
```

Comment:
Let's either remove this or use `@Disabled` if it's temporary one

Reply:
Test reintroduced